src/Sulu/Bundle/ContentBundle/Tests/Features/content_type_multiple_select.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Multiple select content type
+    In order to provide a multiple select field
+    As a user
+    I need to be able to select multiple items in a page
+
+    Background:
+        Given there exists a page template "multiple_select_page" with the following property configuration
+        """
+        <property name="select" type="multiple_select">
+            <meta>
+                <title lang="de">Select</title>
+            </meta>
+            <params>
+                <param name="values" type="collection">
```

Comment:
if someone forgets to set this property param the generation of form fails. therefor you should provide a default parameter in the content type